easybuild/easyconfigs/c/CESM/CESM-2-iomkl-2018b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'Bundle'
+
+name = 'CESM'
+version = '2'
+
+homepage = 'http://www.cesm.ucar.edu/models/cesm2/'
+description = """CESM is a fully-coupled, community, global climate model that provides state-of-the-art
+ computer simulations of the Earth's past, present, and future climate states."""
+
+toolchain = {'name': 'iomkl', 'version': '2018b'}
+
+dependencies = [
+    ('CMake', '3.11.4'),
+    ('Python', '2.7.15'),
+    ('Perl', '5.28.0'),
+    ('ESMF', '7.1.0r'),
+    ('Subversion', '1.9.9'),
+    ('netCDF', '4.6.1'),
+    ('git', '2.18.0'),
+    ('netCDF-Fortran', '4.4.4'),
+    ('netCDF-C++4', '4.3.0'),
+]
+
+modloadmsg = """Now, download the code to a designated directory, for example:
+git clone -b release-cesm2.0.1 https://github.com/ESCOMP/cesm.git
+then edit the versions of external programs in cesm/External.cfg (if needed)
+then download the external programs:
+cd cesm; ./manage_externals/checkout_externals
+then checkout the desired branch of external programs (if needed).
```

Comment:
@boegel `geo` is the new `bio`. I was discussing with the users, and they confirmed that they might have to change locally their software or checkout different versions. W.R.T. the externals, they might have to change the version, or checkout different branches (here it is svn and git mixed), they might change the externals (depends on how complicated the weather model). Then you have to make a machine-file (only once) with paths, compilers, needed modules, queue system (yes, the program writes submit scripts!!!). After versioning (and the machine file) they have to compile the model (yes, they have to compile before the simulation, and after they set up the simulation and some parameters like grid, etc.). Then they have to download the inputs (only those parts what they need, because all the input data right now is 10Tb, so not recommended to download the whole). The you can submit the script what the program wrote. Plus they are in transition from v1 to v2 and from svn to git. Documentation in 3 different places. Honestly, I am not sure what more we could do in EasyBuild. (see https://escomp.github.io/cesm/release-cesm2/downloading_cesm.html , https://github.com/ESCOMP/cesm , https://escomp.github.io/cesm/release-cesm2/introduction.html )

Reply:
@vanzod Thoughts here? It indeed seems there isn't much more we can do here, other than try to provide all dependencies for CESM.

Maybe we do need to rename this bundle to something else though, since it's not actually `CESM` itself? Maybe `CESM-deps`, and make the version a datestamp @hajgato?

Reply:
@boegel @vanzod Indeed, maybe we just have to rename it to `CESM-deps`, although the compiler/mpi combo matters. So maybe `CESM/2-intel-2018b-deps` or `CESM2/intel-2018b-deps` ?

Reply:
@hajgato Sorry for not getting back to this...

`CESM2-deps` makes more sense to me, since this bundle only provided dependencies for `CESM2`, not `CESM2` itself...